src/EnergyPlus/DataLoopNode.hh

```diff
@@ -173,6 +173,10 @@ namespace DataLoopNode {
 		Real64 GenContamSetPoint; // {ppm}
 		bool SPMNodeWetBulbRepReq; // Set to true when node has SPM which follows wetbulb
 
+		int  WindCoefCurveNum; // Wind coefficient Curve ID, replace with pointer after curve refactor
```

Comment:
Does the node really need to know the pressure coefficient curve?  Maybe it does, but I have concerns that the logic that checks for this elsewhere is going to get tripped up by the automatic curve generation that takes place elsewhere.

Reply:
You're right. It probably does not. I changed the logic a bit to read this in the structure MultiZoneExternalNode only and added some checks.